server/src/main/java/io/druid/segment/loading/SegmentLoader.java

```diff
@@ -32,4 +33,5 @@
   public Segment getSegment(DataSegment segment) throws SegmentLoadingException;
   public File getSegmentFiles(DataSegment segment) throws SegmentLoadingException;
   public void cleanup(DataSegment segment) throws SegmentLoadingException;
+  public List<File> getStorageLocations();
```

Comment:
this method probably does not make , not all implementations need to have a sense of local storageLocations as SegmentLoaderLocalCacheManager does.
I think it has been added to access those in ZkCoordinator. is it possible to get same directly from SegmentLoaderConfig reference available in ZkCoordinator given that it already assumes that SegmentLoader in use is the SegmentLoaderLocalCacheManager ?